### PR TITLE
[DoNotMerge]Nasingla/newversion

### PR DESCRIPTION
**Pull request by navysingla**
We are migrating our legacy code from Cloud service to Web app. In this migration, we are also consolidating all our APIs into one version - 2019-10-01-preview as right now they are spread across multiple versions. It will help us to maintain and build SDKs, documentation and Azure CLI easily. Functionally there won't be any change in APIs

### CODE REVIEW

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]
Please refer this new rp file in `readme.md` file, which will resolve the failure in Avacado

**Comment by navysingla** [comment]
This file is not mentioned in readme.md file on purpose as we don't want to build SDK and add documentation for this new version right now.

**Comment by ArcturusZhang** [comment]
> This file is not mentioned in readme.md file on purpose as we don't want to build SDK and add documentation for this new version right now.

Then maybe you should add some `exclude files` entries to bypass the Avacado failure.

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/enable": {
```
Comment (KrisBash) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json]: Are more operations coming? Should we hold review for all operations or review as is?

Comment (navysingla) [reply specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json]: Yes we will add more operations but we are not sure about their timeline yet. For now we want to make cancel,enable,rename prod ready

**Comment by navysingla** [comment]
> This file is not mentioned in readme.md file on purpose as we don't want to build SDK and add documentation for this new version right now.
> 
> If you do not want to build SDK, you can leave the code generation part empty. But you must add this file to input-file section in a new tag or existed tag to avoid the CI failure.

Reply - We want to generate SDK but from older versions not from the new one, how can we do that?

**Comment by ArcturusZhang** [comment]
> > This file is not mentioned in readme.md file on purpose as we don't want to build SDK and add documentation for this new version right now.
> > If you do not want to build SDK, you can leave the code generation part empty. But you must add this file to input-file section in a new tag or existed tag to avoid the CI failure.
> 
> Reply - We want to generate SDK but from older versions not from the new one, how can we do that?

Add a new section in `readme.md` file, such as

```
### Tag: package-2019-10-preview

These settings apply only when `--tag=package-2019-10-preview` is specified on the command line.

``` yaml $(tag) == 'package-2019-10-preview'
input-file:
- Microsoft.Subscription/preview/2019-10-01-preview/subscriptionDefinitions.json
```
```

No SDK will be generated because this tag is not referenced in code-generation part.

**Comment by navysingla** [comment]
> This file is not mentioned in readme.md file on purpose as we don't want to build SDK and add documentation for this new version right now.
> If you do not want to build SDK, you can leave the code generation part empty. But you must add this file to input-file section in a new tag or existed tag to avoid the CI failure.
> 
> Reply - We want to generate SDK but from older versions not from the new one, how can we do that?
> 
> Add a new section in readme.md file, such as
> ### Tag: package-2019-10-preview
> 
> These settings apply only when `--tag=package-2019-10-preview` is specified on the command line.
> 
> ``` yaml $(tag) == 'package-2019-10-preview'
> input-file:
> - Microsoft.Subscription/preview/2019-10-01-preview/subscriptionDefinitions.json
> 
> 
> No SDK will be generated because this tag is not referenced in code-generation part.

Updated readme.md file

**Comment by navysingla** [comment]
> /azp run

Did not understand?

**Comment by ArcturusZhang** [comment]
> > /azp run
> 
> Did not understand?

This is a command to make the CIs to restart. Please ignore this.

**Comment by navysingla** [comment]
Can you approve it ??

**Comment by pilor** [comment]
@navysingla This only changes the operationId of the 3 operations in this api-version.  These operations were already present in 2019-03-01-preview.  If you aren't generating code why is this new swagger spec being checked in?

**Comment by navysingla** [comment]
> @navysingla This only changes the operationId of the 3 operations in this api-version.  These operations were already present in 2019-03-01-preview.  If you aren't generating code why is this new swagger spec being checked in?

We are migrating our legacy code from Cloud service to Web app. In this migration, we are also consolidating all our APIs into one version - 2019-10-01-preview as right now they are spread across multiple versions. It will help us to maintain and build SDKs, documentation and Azure CLI easily. Functionally there won't be any change in APIs

**Review by pilor — APPROVED** [review]

**Review by ArcturusZhang — APPROVED** [review]

**Comment by ArcturusZhang** [comment]
Hi @navysingla please let me know when this is ready to merge. You can remove the "DoNotMerge" from title then.

**Comment by ArcturusZhang** [comment]
Temporary close due to no response from the author. Feel free to reopen when you are ready to merge @navysingla

**Pull request merged**